saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/web/Saml2AuthenticationTokenConverter.java

```diff
@@ -124,12 +127,59 @@ private String samlInflate(byte[] b) {
 			InflaterOutputStream inflaterOutputStream = new InflaterOutputStream(out, new Inflater(true));
 			inflaterOutputStream.write(b);
 			inflaterOutputStream.finish();
-			return out.toString(StandardCharsets.UTF_8.name());
+			return out.toString(StandardCharsets.UTF_8);
```

Comment:
Thanks for this cleanup. Will you please place it in a separate commit so that it can be easily backported?

Reply:
Done